great_expectations/core/factory/validation_factory.py

```diff
@@ -0,0 +1,56 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+from great_expectations._docs_decorators import public_api
+from great_expectations.compatibility.typing_extensions import override
+from great_expectations.core.factory.factory import Factory
+
+if TYPE_CHECKING:
+    from great_expectations.core.validation import Validation
+
+
+# TODO: Add analytics as needed
+class ValidationFactory(Factory[Validation]):
+    def __init__(self, store) -> None:
+        # TODO: Update type hints when new ValidationStore is implemented
+        self._store = store
```

Comment:
Think I'm blocked on this. We have an existing `ValidationsStore` but those are for results?

I think we need to do the following:
- Rename the old store to `ValidationResultsStore`
- Create a new `ValidationsStore` (or `ValidationStore` our inconsistent plurality should be resolved)
- Plug in the new one here and have CRUD start working

Reply:
Working on the store now: https://github.com/great-expectations/great_expectations/pull/9515